nix/postgres.nix

```diff
@@ -136,6 +136,22 @@ in
           '';
         };
 
+        defaultSettings =
+          lib.mkOption {
+            type = lib.types.raw;
```

Comment:
Shouldn't this be `attrsOf (oneOf [ bool float int str ]);`?

Reply:
Since it was `internal` I thought it's fine to be raw, but doesn't hurt to be more specific.